psa-crypto/src/types/status.rs

```diff
@@ -19,6 +19,16 @@ pub enum Status {
     Error(Error),
 }
 
+impl Status {
+    /// Convert the Status into a Result returning the empty tuple
+    pub fn to_result(self) -> Result<()> {
+        match self {
+            Status::Success => Ok(()),
+            Status::Error(error) => Err(error),
+        }
+    }
+}
```

Comment:
Why not implementing `From<Status> for Result`?

Reply:
I will add it as well! I am keeping both as in some contexts when the compiler can not infer the type it might be more convenient to use `...to_result()` rather than `Result<()>::from(...)`